tests/dhcp_relay/test_dhcp_relay_stress.py

```diff
@@ -95,3 +98,133 @@ def _check_socket_buffer():
+            duthost=duthost, interface=client_port_name,
+            pkts_filter="ether src %s and udp dst port %s" % (client_mac, DEFAULT_DHCP_SERVER_PORT),
+            pkts_validator=verify_server_packets
+        ):
+            ptf_runner(ptfhost, "ptftests", "dhcp_relay_stress_test.DHCPStressDiscoverTest",
+                       platform_dir="ptftests", params=params,
+                       log_file="/tmp/dhcp_relay_stress_test.DHCPStressTest.log",
+                       qlen=100000, is_python3=True, async_mode=True)
+            check_dhcp_stress_status(duthost, packets_send_duration)
+            exp_count = int(ptfhost.shell('cat /tmp/dhcp_stress_test_discover.json')['stdout'].strip())
+
+        with capture_and_check_packet_on_dut(
+            duthost=duthost, interface=server_port_name,
+            pkts_filter="ether src %s and udp dst port %s" % (server_mac, DEFAULT_DHCP_SERVER_PORT),
+            pkts_validator=verify_client_packets
+        ):
+            ptf_runner(ptfhost, "ptftests", "dhcp_relay_stress_test.DHCPStressOfferTest",
+                       platform_dir="ptftests", params=params,
+                       log_file="/tmp/dhcp_relay_stress_test.DHCPStressTest.log",
+                       qlen=100000, is_python3=True, async_mode=True)
+            check_dhcp_stress_status(duthost, packets_send_duration)
+            exp_count = int(ptfhost.shell('cat /tmp/dhcp_stress_test_offer.json')['stdout'].strip())
+
+        with capture_and_check_packet_on_dut(
+            duthost=duthost, interface=client_port_name,
+            pkts_filter="ether src %s and udp dst port %s" % (client_mac, DEFAULT_DHCP_SERVER_PORT),
+            pkts_validator=verify_server_packets
+        ):
+            ptf_runner(ptfhost, "ptftests", "dhcp_relay_stress_test.DHCPStressRequestTest",
+                       platform_dir="ptftests", params=params,
+                       log_file="/tmp/dhcp_relay_stress_test.DHCPStressTest.log",
+                       qlen=100000, is_python3=True, async_mode=True)
+            check_dhcp_stress_status(duthost, packets_send_duration)
+            exp_count = int(ptfhost.shell('cat /tmp/dhcp_stress_test_request.json')['stdout'].strip())
+
+        with capture_and_check_packet_on_dut(
+            duthost=duthost, interface=server_port_name,
+            pkts_filter="ether src %s and udp dst port %s" % (server_mac, DEFAULT_DHCP_SERVER_PORT),
+            pkts_validator=verify_client_packets
+        ):
+            ptf_runner(ptfhost, "ptftests", "dhcp_relay_stress_test.DHCPStressAckTest",
+                       platform_dir="ptftests", params=params,
+                       log_file="/tmp/dhcp_relay_stress_test.DHCPStressTest.log",
+                       qlen=100000, is_python3=True, async_mode=True)
+            check_dhcp_stress_status(duthost, packets_send_duration)
+            exp_count = int(ptfhost.shell('cat /tmp/dhcp_stress_test_ack.json')['stdout'].strip())
```

Comment:
ptfrunner run in async mode, would this case end directly when execute to this line without ptfrunner end?